src/Menu.js

```diff
@@ -24,13 +24,26 @@ export class Menu extends Component {
     }
   }
 
+  // Wrap the MenuProvider's handler in a local function so we can unregister it for
+  // one Menu instance without affecting other instances
+  _handleBackButton = () => this.props.ctx.handleBackButton();
+
   componentDidMount() {
     if (!this._validateChildren()) {
       return;
     }
     debug('subscribing menu', this._name);
     this.props.ctx.menuRegistry.subscribe(this);
     this.props.ctx.menuActions._notify();
+
+    if (BackHandler) {
+      BackHandler.addEventListener('hardwareBackPress', this._handleBackButton);
+    } else {
+      const { backHandler } = this.props;
```

Comment:
this won't work - since there is no such prop - but easy way is to keep the check in provider (so that warning is logged only once and provider has the prop)  and let menu do only registration.

Reply:
Good catch! Surprised I missed that. Your suggestion makes sense, however I must again ask, do you think the check is worth keeping? React Native 0.44 is verrrrrry old, and while I could believe apps using it may still exist, I find it hard to believe that any app still using that would be updating to the latest version of this package.

And the README already states:

popup-menu version | min RN (React) version
-- | --
0.13 | 0.55 (16.3.1)

Reply:
you are right, it can be removed

Reply:
Removed. :)